src/Microsoft.DotNet.VersionTools/project.json

```diff
@@ -2,8 +2,8 @@
   "dependencies": {
     "NETStandard.Library": "1.6.0",
     "Newtonsoft.Json": "9.0.1",
-    "NuGet.Packaging": "4.3.0",
-    "NuGet.Versioning": "4.3.0",
+    "NuGet.Packaging": "4.4.0",
+    "NuGet.Versioning": "4.4.0",
```

Comment:
This needs to be updated in the `Microsoft.DotNet.VersionTools` nuspec as well:
https://github.com/dotnet/buildtools/blob/c74c125e5939f8ed964e37f28302d4e3569a9890/src/nuget/Microsoft.DotNet.VersionTools.nuspec#L21-L22

https://github.com/dotnet/buildtools/issues/1886 tracks pulling these versions from `dependencies.props` for the nuspec.

Reply:
oh sorry I missed that one, I'll fix this.